tests/aws/services/s3/test_s3_api.py

```diff
@@ -623,6 +624,47 @@ def test_upload_part_copy_range(self, aws_client, s3_bucket, snapshot):
                 )
             snapshot.match(f"upload-part-copy-range-exc-{src_range}", e.value.response)
 
+    @markers.aws.validated()
+    @pytest.mark.xfail(
+        condition=config.LEGACY_V2_S3_PROVIDER,
+        reason="Moto does not handle the exceptions properly",
+    )
```

Comment:
this test works against the `legacy_v2` provider, I've just tried it. However, we need to add a `skip_snapshot_verify` marker on the class above, I've commented it. We can remove the `xfail`.

Reply:
removed xfail